src/Shared/HashCodeCombiner/HashCodeCombiner.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
Can we get rid of the usages of this class in favor of the one in the framework?

Reply:
Probably in mvc.